Kconfig.zephyr

```diff
@@ -318,6 +318,25 @@ config BUILD_OUTPUT_BIN
 	  Build a "raw" binary zephyr/zephyr.bin in the build directory.
 	  The name of this file can be customized with CONFIG_KERNEL_BIN_NAME.
 
+config BUILD_OUTPUT_UF2
```

Comment:
Same as with CMake code.
Should be part of the module, and thus be in `modules/Kconfig.uf2`.